tools/manifest-templator/manifest-templator.go

```diff
@@ -260,7 +260,7 @@ func getCDI(data *templateData) {
 	// Get CDI Deployment
 	cdideployment := cdicomponents.GetDeployment(
 		"kubevirt",
-		"v1.7.0",
+		"v1.9.0",
```

Comment:
Should also run `./hack/build-manifests.sh` and include those changes in this PR.

Reply:
@djzager 
done:
now pr contains  genarated manifests:
```
deploy/converged/olm-catalog/kubevirt-hyperconverged/0.0.1/kubevirt-hyperconverged-operator.v0.0.1.clusterserviceversion.yaml
deploy/converged/operator.yaml

```